src/utils/components/Consoles/components/CloudInitCredentials/CloudInitCredentialsContent.tsx

```diff
@@ -4,19 +4,19 @@ import { Trans } from 'react-i18next';
 import { V1VirtualMachineInstance } from '@kubevirt-ui/kubevirt-api/kubevirt';
 import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
 import { getCloudInitCredentials } from '@kubevirt-utils/resources/vmi';
+import { isEmpty } from '@kubevirt-utils/utils/utils';
 import { ClipboardCopy } from '@patternfly/react-core';
 
-import { CLOUD_INIT_MISSING_USERNAME } from '../../utils/constants';
-
 type CloudInitCredentialsContentProps = {
   vmi: V1VirtualMachineInstance;
 };
 
 const CloudInitCredentialsContent: React.FC<CloudInitCredentialsContentProps> = ({ vmi }) => {
   const { t } = useKubevirtTranslation();
-  const { user, password } = getCloudInitCredentials(vmi);
+  const { users } = getCloudInitCredentials(vmi);
+  const usernameTitle = users?.length > 1 ? t('User names: ') : t('User name: ');
```

Comment:
i think we can use the auto plurals from i18n , some reference: https://www.i18next.com/translation-function/plurals

the result should be something like this:
<img width="549" alt="image" src="https://user-images.githubusercontent.com/14824964/183284254-658ae901-c4de-46d0-be68-3c85d6d36d17.png">

<img width="487" alt="image" src="https://user-images.githubusercontent.com/14824964/183284227-da3f4817-ea8d-4a78-9a9e-3b11653c6907.png">